README.md

```diff
@@ -165,6 +203,47 @@ $config = [
 ACFComposer\ACFComposer::registerFieldGroup($config);
 ```
 
+It will result in two wysiwygs being displayed but those would have the same name and thus overwrite each other's data.
+
+In order to fix this, there is the notion of filter arguments. These have to be appended with a `#` to the filter name. Once this is done, the respective filter will be called with the suffix as a second argument, and the field names will be prefixed with that string.
+
+Rewriting the previous example:
```

Comment:
For example, taking the previous broken code an adding two unique filter names will resolve the problem: